php/libraries/SinglePointLogin.class.inc

```diff
@@ -253,6 +253,11 @@ class SinglePointLogin
 
         $this->_lastError = '';
 
+        $factory = NDB_Factory::singleton();
+        $config  = $factory->config();
+
+        $maxDaysInactive = intval($config->getSetting("UserMaximumDaysInactive"));
+
```

Comment:
not sure if we want to throw an exception if this value is not set? like do we have HIPAA regulations where an account has to at some point expire?